htdocs/view/layout/header.php

```diff
@@ -104,19 +104,32 @@
   </li>
 
   <li>
-		<?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
+    <?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
   </li>
 
   <!-- Modal : the user can choose the wave to ask for subsidies (make a request) -->
   <?php
     if (isset($_GET["prefix"]) && $_GET["prefix"] == "binet" && has_editing_rights($binet, $term)) {
         ob_start();
-        $waves_for_modal = select_waves(array("state" => "submission"), "submission_date", false);
+        $waves_for_modal = select_waves(array("state" => array("submission", "deliberation")), "submission_date", false);
```

Comment:
Non j'ai pas testé je te faisais confiance à vrai dire...
On ne pourrait pas carrément tester si on cherche à parcourir un array et du coup rajouter le IN ?